frontend/package.json

```diff
@@ -51,6 +51,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "cy:run": "cypress run --headless --browser chrome",
+    "cy:open": "cypress open --browser chrome"
+
```

Comment:
> @manzil-infinity180
> 
> just a suggestion what if chrome is not present in local machine maybe you can configure the script such that it opens in default browser of the local machine
> 
> but great 🚀, this PR enables ease of testing

it will automatically open into the electron (i think this was the default one)
and also it use electron to run the script in terminal